lms/services/digest/_models.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+from functools import cached_property
+from typing import List
+
+
+@dataclass
+class HUser:
+    """A class representing H users."""
+
+    h_userid: str
+    """The H userid like 'acct:name@lms.example.com'."""
+
+    display_name: str
+    """A human readable display name."""
+
+    email: str
+    """Email for the user."""
```

Comment:
Having all these fields mandatory feels correct for the actual code, but sucks for the tests. We often have to fill out loads of fields we don't care about to get a fixture.

I'm not sure which is worse?